src/components/screensavermanager.js

```diff
@@ -18,6 +20,37 @@ define(["events", "playbackManager", "pluginManager"], function (events, playbac
         }
     });
 
+    var isLoggedIn;
+    var defaultOption = isLoggedIn ? "backdropscreensaver" : "logoscreensaver";
```

Comment:
this won't work, as you determine the value of `isLoggedIn` down below

Reply:
https://github.com/jellyfin/jellyfin-web/pull/592/commits/1a63de77e16d59abc9bff889f1007a0b69fa8f91